tests/phpunit/CRM/Batch/Form/EntryTest.php

```diff
@@ -117,26 +122,26 @@ public function setUp(): void {
       'domain_id' => CRM_Core_Config::domainID(),
     ];
     $membershipType = $this->callAPISuccess('membership_type', 'create', $params);
-    $this->_membershipTypeID = $membershipType['id'];
+    $this->_membershipTypeId = $membershipType['id'];
 
-    $this->_orgContactID2 = $this->organizationCreate();
+    $this->_orgContactId2 = $this->organizationCreate();
```

Comment:
This is the opposite direction to which I have been renaming variables in - ie `ID` is a capitalised abbreviation in English ('Student ID') whereas `id` is something freud was all into.

I would rename `$this->_orgContactId2` to `$this->organizationContactID2;` in order to get rid of the assumption that people (especially non English speaking people) realise that `org` is an abbreviation for `Organization`.  We have also been generally removing the leading underscores except where they are meaningful in the code

Reply:
I also dislike the leading underscores, but thought that was just me. So happy to change those. Will push an update in a moment.